source/sass/buyers-guide/views/product.scss

```diff
@@ -1,5 +1,3 @@
-$pni-product-breakpoint-larger: $bp-md;
```

Comment:
Moved to [source/sass/buyers-guide/variables.scss](https://github.com/mozilla/foundation.mozilla.org/pull/6528/files#diff-e5e5421422d79fcbdb311aed60c81209c8785ca2f292dba3d6dbab4b492030f4)